### ISSUE TITLE
Not able to download Selected Archive & Feature request

### ISSUE THREAD

**Issue by Rupthya** [webrecorder/archiveweb.page#25]
Thanks for your help again, just like the title said.

I just think I can not do this since 0.6.0 (Maybe)

---

Also a question about archive Amazon product page:
![example](https://i.imgur.com/iy0WZvX.png)
https://www.amazon.co.jp/PSP-｢LocoRoco｣オリジナル・サウンドトラック-｢ロコロコのうた｣-ゲーム・ミュージック/dp/B000HEWJK8/

click the link inside the purple box, result in the highlighted with a red box pop up. But I can't figure out how to do this with the extension archive.

**Comment by jakewarren** [comment]
I can replicate the issue of not being able to download selected archives on v0.6.1. 

I receive the following error in the console:
```
ui.js:2072 Uncaught TypeError: Cannot read property 'currentTarget' of undefined
    at HTMLElement.onSendSelToggle (ui.js:2072)
    at v.handleEvent (ui.js:2825)
    at HTMLInputElement.__boundHandleEvent (ui.js:2825)
onSendSelToggle @ ui.js:2072
handleEvent @ ui.js:2825
__boundHandleEvent @ ui.js:2825
```

**Comment by ikreymer** [comment]
The selection issue should be fixed in 0.6.2!

@Rupthya I'm not sure I understand the feature request -- can you open a new issue for that? Is it a capture or replay issue?

**Comment by jakewarren** [comment]
thank you @ikreymer! I can confirm that v0.6.2 does resolve the selection issue on my end.
